src/App.js

```diff
@@ -38,10 +38,10 @@ export function App () {
         init();
     }, []);
 
-    const switchTheme = (theme) => {
-        localStorage.setItem(LOCAL_STORAGE_KEYS.UI_THEME, theme);
-        document.getElementById("app").setAttribute("data-theme", theme);
-        setTheme(theme);
+    const switchTheme = (newTheme) => {
```

Comment:
This isn't related to this change, right? (This is fine for this PR since it's small, but typically we try to limit the blast radius of the changes in a PR so we don't change files unrelated to it, even if it's simple refactoring. It's fine to have a separate PR just for refactoring things.)